### DISCUSSION TITLE
Quarkus Project on Windows native problem

### DISCUSSION THREAD

**Discussion by shivarm** [quarkusio/quarkus#33611] [Community]
I 'm contributing currently two project which are quarkus based and I have see so many issue like- build failure, file system issues, even while set-up project locally I got issue.

My question is why? In other hand Quarkus project works better in Linux and MacOS.

**Comment by maxandersen** [comment]
Hard to help/explain without some specific issues/errors?

Windows has some limitations and trickier config when it comes to native building in general (even outside quarkus and java) and so many different installs and we don't have every combination to test. so I'm not surprised if there are issues.

To fix it - open issues with logs/errors and steps to reproduce and we can help explore and find ways to fix it together.

**Reply by shivarm** [reply]
We have runnig project with using quarkus @jesperpedersen @johnaohara can you explain what we are facing with native windows build with quarkus?

**Reply by johnaohara** [reply]
@shivam-sharma7 I am assuming you are talking about https://github.com/Hyperfoil/Horreum?

We do not build a native image for Horreum, it runs in JVM mode. The issues that you have been facing afaik relate to issues running the Horreum infrastructure in Windows on WSL2, not with Quarkus itself 

The 2 issues that you have faced are https://github.com/Hyperfoil/Horreum/discussions/524 and https://github.com/Hyperfoil/Horreum/discussions/525. Are there any more that I am unaware of?

Jesper is working on https://github.com/Hyperfoil/Horreum/discussions/524 at the moment, and https://github.com/Hyperfoil/Horreum/discussions/525 was a local environment issue with WSL2.

If there are further issues with Horreum, please can you open a discussion at https://github.com/Hyperfoil/Horreum/discussions 

I am not aware of an specific issues with running Horreum in a Windows environment

**Reply by shivarm** [reply]
@johnaohara Thanks for the clarification so I think most error we getting because `OS` is different I am also working on my Quarkus CRUD project which build in Windows 11 and running correct https://github.com/shivam-sharma7/Quarkus-CRUD-Demo

And https://github.com/Hyperfoil/Horreum this build on Linux so I think our issue is with docker `image`? for Windows

Thanks to @maxandersen for the response.
